README.md

```diff
@@ -205,23 +205,24 @@ GoogleMap(
 
 ### Recomposing elements
 
-Markers and other elements need to be recomposed in the screen. To achieve recomposition you need to 
-have a mutable state variable:
+Markers and other elements need to be recomposed in the screen. To achieve recomposition, you can set mutable properties of state objects:
 
 ```kotlin
-var location by remember { mutableStateOf(singapore) }
-val singaporeState = MarkerState(position = location)
+val markerState = remember { MarkerState(position = singapore) }
```

Comment:
Actually, how about `rememberMarkerState()` instead to retain state across configuration changes? (It uses `rememberSaveable()` behind the scenes.)
```suggestion
val markerState = rememberMarkerState(position = singapore)
```